src/map/ai/states/range_state.cpp

```diff
@@ -201,18 +201,6 @@ bool CRangeState::CanUseRangedAttack(CBattleEntity* PTarget, bool isEndOfAttack)
                 break;
             }
             case SKILL_ARCHERY:
-            {
-                PRanged = dynamic_cast<CItemWeapon*>(PChar->getEquip(SLOT_AMMO));
```

Comment:
why is this removed?

Reply:
duplicate code, skill_marksmanship has the exactly same thing